pkg/k8shandler/fluentd.go

```diff
@@ -337,6 +354,26 @@ func (clusterRequest *ClusterLoggingRequest) createOrUpdateFluentdSecret() error
 	return nil
 }
 
+func newLogMetricExporterContainer() v1.Container {
+	resources := v1.ResourceRequirements{}
```

Comment:
@alanconway I left this blank when I provided the PR, but how should we pass in resources here.  I'm not fond of opening up the API, but maybe we need to allow a backdoor for setting mem and cpu, maybe via annotations?

Reply:
We could add a "MetricExporterSpec" to the API, I don't think that would be too bad. The API already has componet-specific requirements (FluentdSpec) so it's not making it any worse. 
Alternatively we could split the Fluentd requirements - e.g. give 90% to fluentd and 10% to the exporter. We'd need to do some benchmarks to make sure that's a reasonable split but we should be doing CPU/Mem benehmarks anyway so it's useful work.
The backdoor approach means our collector is using resources that the user doesn't know about, which is not ideal.

Reply:
I would lean towards just adding to the API. This is something to clean up in the "API v2" effort when we get smarter about scaling and resource use, but it's not worse than whats already there.

Reply:
@pmoogi-redhat this means we need to add soming like https://github.com/openshift/cluster-logging-operator/blob/master/pkg/apis/logging/v1/clusterlogging_types.go#L35  with the resource requirements.  Make sure it additionally includes the comments as these are used for the code generation

Reply:
@jcantrill @alanconway added changes to support LogFileMetricExportedSpec - please review my changes done in 	 
        modified:   apis/logging/v1/clusterlogging_types.go
	modified:   k8shandler/defaults.go
	modified:   k8shandler/fluentd.go
Hoping above is sufficient for specifying individually resources for LogFileMetricExporter.

Reply:
@vimalk78 please review as well